src/images/zookeeper.rs

```diff
@@ -0,0 +1,88 @@
+use crate::core::Port;
+use crate::{Container, Docker, Image, WaitForMessage};
+use std::collections::HashMap;
+
+const CONTAINER_IDENTIFIER: &str = "bitnami/zookeeper";
+const DEFAULT_TAG: &str = "3.5.7";
+
+#[derive(Debug, Default, Clone)]
+pub struct ZookeeperArgs;
+impl IntoIterator for ZookeeperArgs {
+    type Item = String;
+    type IntoIter = ::std::vec::IntoIter<String>;
+
+    fn into_iter(self) -> <Self as IntoIterator>::IntoIter {
+        vec![].into_iter()
+    }
+}
+
+#[derive(Debug)]
+pub struct Zookeeper {
+    tag: String,
+    arguments: ZookeeperArgs,
+    ports: Option<Vec<Port>>,
+}
+
+impl Default for Zookeeper {
+    fn default() -> Self {
+        Zookeeper {
+            tag: DEFAULT_TAG.to_string(),
+            arguments: ZookeeperArgs {},
+            ports: None,
```

Comment:
Does zookeeper have a default port? If yes we can include it here to make usage easier!

Reply:
Hi, @thomaseizinger  sure. Could i separate zookeeper tests in another file?